src/rust/engine/src/externs/process.rs

```diff
@@ -104,7 +108,15 @@ impl PyProcessExecutionEnvironment {
     #[getter]
     fn docker_image(&self) -> Option<&str> {
         match &self.environment.strategy {
-            ProcessExecutionStrategy::Docker(image) => Some(image),
+            ProcessExecutionStrategy::Docker { image, .. } => Some(image),
+            _ => None,
+        }
+    }
+
+    #[getter]
+    fn docker_mounts(&self) -> Option<Vec<String>> {
```

Comment:
Nit-pick: Maybe `docker_mounts` could return `Option<Vec<&str>>` (similar to `docker_image`'s return value) to allow the caller to decide whether it needs a copy or not?